doc/api/n-api.md

```diff
@@ -3134,6 +3134,30 @@ overhead in creating/storing strings with this method.
 The JavaScript `string` type is described in
 [Section 6.1.4][] of the ECMAScript Language Specification.
 
+> **Stability**: 1 - Experimental
```

Comment:
This seems like a strange place to put a stability banner. I would expect it to appear with a section heading (and accompanying YAML) indicating what part of the API it's talking about. See, for example, from earlier in the doc:

```md

#### `node_api_create_property_key_utf16`
<!-- YAML
added:
  - v21.7.0
  - v20.12.0
-->
> Stability: 1 - Experimental
```

Reply:
thank you for suggestions, ı applied your comments